source/fundamentals/bson.txt

```diff
@@ -210,6 +210,29 @@ Driver will marshal structs using the following rules:
       - Includes an empty ``lastname`` field
       - Stores the ``Address`` field as a nested value
 
+BSON Options
+------------
+
+You can set ``BSONOptions`` to specify how
+the driver marshals and unmarshals BSON on any level, from
+``collection`` to ``Client``.
+
+The following example demonstrates how the ``BSONOptions`` is set using
+the ``Movies`` collection as an example.
+
+.. code-block:: go
+   :copyable: false
+
+   bsonOpts := &options.BSONOptions{
+		UseJSONStructTags: true,
+	}
+	coll := client.Database("sample_mflix").Collection("movies",
+	bsonOpts)
+   
+To learn more about the options and usage of ``BSONOptions``, visit the `API Documentation
```

Comment:
S: Put this sentence in a Tip box, since it's more supplementary than a natural conclusion to this section